# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../acl/destdir.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./doc/Makefile.orig	2007-10-15 13:05:47.000000000 -0300
+++ ./doc/Makefile	2007-10-15 13:07:07.000000000 -0300
@@ -17,12 +17,12 @@
 	$(ZIP) --best -c < CHANGES > $@
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 PORTING CHANGES.gz $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 PORTING CHANGES.gz $(DESTDIR)$(PKG_DOC_DIR)
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -S CHANGES.gz $(PKG_DOC_DIR)/changelog.gz
+	$(INSTALL) -S CHANGES.gz $(DESTDIR)$(PKG_DOC_DIR)/changelog.gz
 else
-	$(INSTALL) -m 644 COPYING $(PKG_DOC_DIR)
+	$(INSTALL) -m 644 COPYING $(DESTDIR)$(PKG_DOC_DIR)
 endif
 
 install-dev install-lib:
--- ./Makefile.orig	2007-10-15 13:05:47.000000000 -0300
+++ ./Makefile	2007-10-15 13:07:30.000000000 -0300
@@ -53,8 +53,8 @@
 
 install: default
 	$(SUBDIRS_MAKERULE)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 README $(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 README $(DESTDIR)$(PKG_DOC_DIR)
 
 install-dev: default
 	$(SUBDIRS_MAKERULE)
--- ./chacl/Makefile.orig	2007-10-15 13:05:47.000000000 -0300
+++ ./chacl/Makefile	2007-10-15 13:07:48.000000000 -0300
@@ -16,6 +16,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev install-lib:
--- ./setfacl/Makefile.orig	2007-10-15 13:05:47.000000000 -0300
+++ ./setfacl/Makefile	2007-10-15 13:08:00.000000000 -0300
@@ -17,6 +17,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev install-lib:
--- ./debian/Makefile.orig	2007-10-15 13:05:47.000000000 -0300
+++ ./debian/Makefile	2007-10-15 13:10:17.000000000 -0300
@@ -16,22 +16,22 @@
 
 install: default
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 644 changelog $(PKG_DOC_DIR)/changelog.Debian
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 644 changelog $(DESTDIR)$(PKG_DOC_DIR)/changelog.Debian
 endif
 
 install-dev: default
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 755 -d $(DEV_DOC_DIR)
-	$(INSTALL) -m 644 copyright $(DEV_DOC_DIR)
-	$(INSTALL) -m 644 changelog $(DEV_DOC_DIR)/changelog.Debian
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(DEV_DOC_DIR)
+	$(INSTALL) -m 644 copyright $(DESTDIR)$(DEV_DOC_DIR)
+	$(INSTALL) -m 644 changelog $(DESTDIR)$(DEV_DOC_DIR)/changelog.Debian
 endif
 
 install-lib: default
 ifeq ($(PKG_DISTRIBUTION), debian)
-	$(INSTALL) -m 755 -d $(PKG_DOC_DIR)
-	$(INSTALL) -m 755 -d $(LIB_DOC_DIR)
-	$(INSTALL) -m 644 copyright $(LIB_DOC_DIR)
-	$(INSTALL) -m 644 changelog $(LIB_DOC_DIR)/changelog.Debian
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DOC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(LIB_DOC_DIR)
+	$(INSTALL) -m 644 copyright $(DESTDIR)$(LIB_DOC_DIR)
+	$(INSTALL) -m 644 changelog $(DESTDIR)$(LIB_DOC_DIR)/changelog.Debian
 endif
--- ./getfacl/Makefile.orig	2007-10-15 13:05:47.000000000 -0300
+++ ./getfacl/Makefile	2007-10-15 13:10:36.000000000 -0300
@@ -17,6 +17,7 @@
 include $(BUILDRULES)
 
 install: default
-	$(INSTALL) -m 755 -d $(PKG_BIN_DIR)
-	$(LTINSTALL) -m 755 $(LTCOMMAND) $(PKG_BIN_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_BIN_DIR)
+	$(LTINSTALL) -m 755 $(LTCOMMAND) $(DESTDIR)$(PKG_BIN_DIR)
+
 install-dev install-lib:
--- ./include/buildmacros.orig	2007-09-10 22:00:47.000000000 -0400
+++ ./include/buildmacros	2007-10-15 13:22:45.000000000 -0300
@@ -69,22 +69,22 @@
 ifeq ($(ENABLE_SHARED),yes)
 INSTALL_LTLIB_DEV = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 $(LIBNAME).lai $(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
-	../$(INSTALL) -m 755 -d $(PKG_LIB_DIR); \
-	../$(INSTALL) -T so_base $(LIBNAME).lai $(PKG_LIB_DIR); \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(PKG_LIB_DIR)/$(LIBNAME).a; \
-	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(PKG_LIB_DIR)/$(LIBNAME).la; \
-	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(PKG_DEVLIB_DIR)/$(LIBNAME).so
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).la ; \
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -T so_base $(LIBNAME).lai $(DESTDIR)$(PKG_LIB_DIR); \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).a $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).a; \
+	../$(INSTALL) -S $(PKG_DEVLIB_DIR)/$(LIBNAME).la $(DESTDIR)$(PKG_LIB_DIR)/$(LIBNAME).la; \
+	../$(INSTALL) -S $(PKG_LIB_DIR)/$(LIBNAME).so $(DESTDIR)$(PKG_DEVLIB_DIR)/$(LIBNAME).so
 else
 INSTALL_LTLIB_DEV = $(INSTALL_LTLIB_STATIC)
 endif
 
 INSTALL_LTLIB_STATIC = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
-	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR)
+	../$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_DEVLIB_DIR); \
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(DESTDIR)$(PKG_DEVLIB_DIR)
 
 INSTALL_MAN = \
 	@for d in $(MAN_PAGES); do \
@@ -96,7 +96,7 @@
 				-e 's/\\\f[0-9]//g' -e 's/  / /g;q'`; \
 		do \
 			[ -z "$$m" -o "$$m" = "\\" ] && continue; \
-			t=$(MAN_DEST)/$$m.$(MAN_SECTION); \
+			t=$(DESTDIR)$(MAN_DEST)/$$m.$(MAN_SECTION); \
 			if $$first; then \
 				if $(HAVE_ZIPPED_MANPAGES); then \
 					$(ZIP) -9 -c $$d > $$d.gz; _sfx=.gz; \
@@ -116,8 +116,8 @@
 INSTALL_LINGUAS = \
 	@for l in $(LINGUAS) ""; do \
 		if test -f "$$l.mo" ; then \
-			ldir=$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
+			ldir=$(DESTDIR)$(PKG_LOCALE_DIR)/$$l/LC_MESSAGES; \
 			$(INSTALL) -m 755 -d $$ldir; \
 			$(INSTALL) -m 644 $$l.mo $$ldir/$(PKG_NAME).mo; \
 		fi; \
 	done
--- ./include/Makefile.orig	2007-10-15 13:32:59.000000000 -0300
+++ ./include/Makefile	2007-10-15 13:33:26.000000000 -0300
@@ -17,9 +17,10 @@
 include $(BUILDRULES)
 
 install-dev: default
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)/sys
-	$(INSTALL) -m 644 acl.h $(PKG_INC_DIR)/sys/acl.h
-	$(INSTALL) -m 755 -d $(PKG_INC_DIR)/acl
-	$(INSTALL) -m 644 libacl.h $(PKG_INC_DIR)/acl/libacl.h
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)/sys
+	$(INSTALL) -m 644 acl.h $(DESTDIR)$(PKG_INC_DIR)/sys/acl.h
+	$(INSTALL) -m 755 -d $(DESTDIR)$(PKG_INC_DIR)/acl
+	$(INSTALL) -m 644 libacl.h $(DESTDIR)$(PKG_INC_DIR)/acl/libacl.h
+
 install install-lib:
